### PR TITLE
twin/mod.rs: update_twin now updates the twin properties

### PR DESCRIPTION
**Pull request by Elleppi**

### CODE REVIEW

File: src/twin/mod.rs [hunk 1]
```diff
@@ -199,9 +199,12 @@ pub async fn update_twin(
         twin_id: Some(twin_id.clone()),
     };
 
+    let property_keys = properties.clone().into_iter().map(|p| p.key).collect();
```
Comment (mihai-dinculescu) [inline src/twin/mod.rs:202]: Should this be computed only when `cleared_all` is `false`?

Comment (Elleppi) [reply src/twin/mod.rs:202]: We update only a small set of properties defined within the `ConnectorData`'s. Anyway if `cleared_all` is `True` this procedure won't make much sense, so I'll update it

Comment (Elleppi) [reply src/twin/mod.rs:202]: However I can see [here](https://github.com/Iotic-Labs/iotics-grpc-client-rust/blob/main/src/client/iotics.api.rs#L1297) that "This operation is ignored if clearAll is True". Do you think I should go ahead and make the change anyway ?

Comment (mihai-dinculescu) [reply src/twin/mod.rs:202]: Let's keep it as it is then!

**Review by mihai-dinculescu — APPROVED** [review]

**Pull request merged**